### PR TITLE
Register quota and credentials for serverless-aws cluster profile

### PR DESCRIPTION
**Pull request by mgencur**
Following the instructions in https://docs.ci.openshift.org/docs/how-tos/adding-a-cluster-profile/#adding-a-new-cluster-profile

Related to https://issues.redhat.com/browse/SRVCOM-2903

/hold

### CODE REVIEW

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -3961,6 +3961,42 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-aws-3
     namespace: ci
+- from:
+    .awscred:
+      field: .awscred
+      item: serverless-aws-credentials
```
Comment (mgencur) [inline core-services/ci-secret-bootstrap/_config.yaml:3967]: I'm not exactly sure what these items correspon to. I have followed instructions from https://docs.ci.openshift.org/docs/how-tos/adding-a-cluster-profile/#providing-credentials and created a secret collection named "serverless-aws-credentials" with keys like in the picture below:
image

**Comment by mgencur** [comment]
This [Slack conversation](https://redhat-internal.slack.com/archives/CBN38N3MW/p1708421403442299) helped resolve issues on the PR.

**Review by jmguzik — APPROVED** [review]

**Pull request merged**